src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClient/SqlColumnEncryptionCertificateStoreProvider.Windows.cs

```diff
@@ -482,10 +482,11 @@ private byte[] RSAEncrypt(byte[] plainText, X509Certificate2 certificate)
         }
 
         /// <summary>
-        /// Encrypt the text using specified certificate.
+        /// Decrypt the text using specified certificate.
```

Comment:
the comment is a little strange, I think it should either be "plaintext" or "data" that it refers to because the return is a byte[] not a text string.

Reply:
Changed to "data".